packages/@o3r/schematics/src/utility/loaders.ts

```diff
@@ -127,7 +127,7 @@ export function getFilesInFolderFromWorkspaceProjectsInTree(tree: Tree, folderIn
   const workspace = readAngularJson(tree);
   const projectSources = Object.values(workspace.projects)
     .map((project) => path.posix
-      .join(project.root, folderInProject, '**', `*.${extension}`)
+      .join(project.root, folderInProject, '**', `*.${extension}$`)
```

Comment:
```suggestion
      .join(project.root, folderInProject, '**', `*\\.${extension}$`)
```
I don' t know if there is any file extension ending with `ts` that is not `.ts` though :)